src/QuadForm/Herm/Mass.jl

```diff
@@ -292,7 +292,7 @@ function _standard_mass(L::HermLat, prec::Int = 10)
   local relzeta::arb
 
   while true
-    relzeta = prod(arb[_L_function(E, 1 - i, wprec) for i in 1:2:m])
+    relzeta = prod(_L_function(E, 1 - i, wprec) for i in 1:2:m; init = 1)
```

Comment:
```suggestion
    relzeta = prod(_L_function(E, 1 - i, wprec) for i in 1:2:m; init = one(ArbField(wprec, cached = false)))
```

Reply:
Ah thanks, that was the thing I was trying to get but couldn't make it work.